stable/spinnaker/templates/statefulsets/halyard.yaml

```diff
@@ -27,8 +27,8 @@ spec:
       serviceAccountName: {{ template "spinnaker.fullname" . }}-halyard
       {{- end }}
       securityContext:
-        runAsUser: 1000
-        fsGroup: 1000
+        runAsUser: {{ .Values.securityContext.userRights }}
```

Comment:
please set these variable names to be `runAsUser` and `fsGroup` to match the `securityContext` keys.

Reply:
done